docs/🤩 Resources/Roadmap to learn Blockchain development.md

```diff
@@ -1,4 +1,14 @@
-Full Roadmap to learn Blockchain development 
+**Full Roadmap to learn Blockchain development**
+
+Click below to see the video
+
 <iframe width="800" height="400" src="https://www.youtube.com/embed/ci_AIMCF-HA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
I think we should remove this. It doesn't even show up well on Markdown.

![image](https://user-images.githubusercontent.com/51391473/196974134-a1fdc40f-22d8-4197-a1c8-1d1e60728e1c.png)

Reply:
Actually, that was already included in the README, and it's YouTube image has been corrupt so cannot display as we did for others. Will remove this as you say.

Reply:
Commented it, tell me if I should delete all the commented content from all the files. Thank you.